dag.yml

```diff
@@ -1,10 +1,19 @@
 steps:
   data://garden/owid/latest/covid:
     - etag://raw.githubusercontent.com/owid/covid-19-data/master/public/data/owid-covid-data.csv
+  data://garden/owid/latest/population_density:
+    - data://garden/owid/latest/key_indicators
+    - data://meadow/open_numbers/latest/open_numbers__world_development_indicators
+    - data://reference
   data://garden/owid/latest/key_indicators:
     - data://meadow/hyde/2017/baseline
     - data://meadow/gapminder/2019-12-10/population
+    # NOTE: it would be better to use /garden/wpp/2019/standard_projections, but it uses different harmonization than gapminder and hyde.
+    # in the future we should build `key_indicators` only from garden datasets
+    - data://meadow/wpp/2019/standard_projections
     - data://reference
```

Comment:
I realised now that all three of these tables would ideally have been harmonized into Garden before being combined here. I'll have a quick go at Hyde baseline now.